### PR TITLE
Translate Ruby 2.2 EOL post (pt)

### PR DESCRIPTION
**Pull request by BannerGames**
I translated the last article to portuguese.
I will stay translating articles when a new is released.
The articles in Portuguese are too old.

### CODE REVIEW

**Comment by stomar** [comment]
CI failed - trailing whitespace needs to be removed.

Also, it should be a single commit, and the commit message improved.

**Comment by jcserracampos** [comment]
@stomar What procedure to take with this PR?

**Comment by stomar** [comment]
First option would be waiting for fixes by @BannerGames. Or, if the language part is ok, I could do the "technical" fixes myself (I can not review the language).

File: pt/news/_posts/2018-06-20-suporte-do-ruby-2-2-acabou.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+layout: news_post
+title: "O suporte do Ruby 2.2 acabou"
+author: "antonpaisov"
+translator: "bannergames"
+date: 2018-06-20 00:00:00 +0000
+lang: pt
+---
+
+Anunciamos que todo o suporte das séries 2.2 do Ruby acabou.
```
Comment (jcserracampos) [inline pt/news/_posts/2018-06-20-suporte-do-ruby-2-2-acabou.md]: Series is singular, so you need to use **da série**.

File: pt/news/_posts/2018-06-20-suporte-do-ruby-2-2-acabou.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+Após o lançamento do Ruby 2.2.7 em 28 de março de 2017,
+o suporte das séries 2.2 do Ruby estava em fase de manunteção de segurança.
```
Comment (jcserracampos) [inline pt/news/_posts/2018-06-20-suporte-do-ruby-2-2-acabou.md]: **da série**

File: pt/news/_posts/2018-06-20-suporte-do-ruby-2-2-acabou.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+Agora, após um ano, essa fase acabou.
+Assim sendo, em 31 de março de 2018, todo o suporte das séries 2.2 do Ruby acabou.
```
Comment (jcserracampos) [inline pt/news/_posts/2018-06-20-suporte-do-ruby-2-2-acabou.md]: **da série**

File: pt/news/_posts/2018-06-20-suporte-do-ruby-2-2-acabou.md (hunk 2 above)
Comment (alaxalves) [reply pt/news/_posts/2018-06-20-suporte-do-ruby-2-2-acabou.md]: @jcserracampos It says "manunteção", it should be "manutenção".

**Comment by stomar** [comment]
I‘ll apply the fixes.

**Comment by stomar** [comment]
Fixes applied and merged. Thanks to all!

**Pull request closed without merging**